opennms-base-assembly/src/main/filtered/etc/opennms.properties

```diff
@@ -869,3 +869,4 @@ org.opennms.newts.nan_on_counter_wrap=true
 
 # By default hostnames are determined for a node's IP addresses during the provisioning's audit phase.
 # org.opennms.provisiond.reverseResolveRequisitionIpInterfaceHostnames=true
+flow.templates.dir=${flow.templates.dir}
```

Comment:
Please also add a comment and describe what this property is used for. Please also add a description of the new options in docs/modules/releasenotes/pages/whatsnew.adoc.